example/ML-KEM-1024-expanded.priv.txt

```diff
@@ -0,0 +1,108 @@
+SEQUENCE {
+  INTEGER { 0 }
+  SEQUENCE {
+    OBJECT_IDENTIFIER { 2.16.840.1.101.3.4.4.3 }
+  }
+  SEQUENCE {
```

Comment:
same here, that should be OCTET STRING

Reply:
[726ca33](https://github.com/lamps-wg/kyber-certificates/pull/100/commits/726ca33b904c2de541dd44617662fb9971a57ffa)